tasks/build.py

```diff
@@ -361,13 +361,16 @@ def create_pr_comments(job, job_id, app_name, job_comment, pr, repo_name, gh, sy
     dt = datetime.now(timezone.utc)
 
     # construct initial job comment
-    job_comment = (f"New job on instance `{app_name}`"
-                   f" for architecture `{arch_name}`"
-                   f" in job dir `{symlink}`\n"
+    comments = config.read_config()["submitted_job_comments"]
```

Comment:
Wonder a bit if `comments` is a good name for the variable. It's more like `submit_comment_cfg` ???